app/controllers/users_controller.rb

```diff
@@ -1,9 +1,17 @@
 class UsersController < ApplicationController
+  before_action :set_user, only: [:show]
+
   def index
     @users = User.all
   end
 
   def show
+    # @user = User.find(params[:id])
+  end
```

Comment:
- [ ] In this activity we are required to use the `recent_post` method defined in the user model, in your case `return_recent_posts`, to render the user's 3 most recent post in the `app/views/users/show.html.erb` view. However, your show action users controller is empty. Kindly improve upon this so that your show action looks like:

```ruby
# app/controllers/users_controller.rb
[...]

def show
  @user = User.find(params[:id])
end

[...]
```